### DISCUSSION TITLE
Is the 2.4.4 documentation missing sections?

### DISCUSSION THREAD

**Discussion by skamansam** [vuetifyjs/vuetify#13162] [Q&A]
I opened up the documentation a few days ago, and received a message saying there were updates. Upon reloading, it appears huge chunks of the documentation are now missing. This is only for the `next` version, at 2.4.4. I wasn't sure if this was intentional and vuetify is moving these things to new repos or what.

![image](https://user-images.githubusercontent.com/102971/109226890-74e21600-778d-11eb-8ee2-067dfec1a99b.png)
![image](https://user-images.githubusercontent.com/102971/109226937-862b2280-778d-11eb-8cf5-11fdf720bf5f.png)

**Comment by KaelWD** [comment]
https://next.vuetifyjs.com/ is documentation for the next branch (v3). Use https://vuetifyjs.com/ for v2.4.4
